### PR TITLE
[MXNET-14075] repair example, enable fault tolerant build of R reference manual

### PR DESCRIPTION
**Pull request by shadogray**
## Description ##
R reference manual shall be built by build process. Due to error in comment the build fails. 

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [X] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [X] Changes are complete (i.e. I finished coding on this PR)
- [X] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [X] R  Reference Manual is built, if LaTex tools are installed and configured

### CODE REVIEW

**Comment by vandanavk** [comment]
@mxnet-label-bot add [R, pr-awaiting-review]

@ankkhedia @anirudhacharya for review

File: Makefile [hunk 1]
```diff
@@ -619,8 +620,19 @@ rpkg:
 	echo "import(Rcpp)" >> R-package/NAMESPACE
 	R CMD INSTALL R-package
 	Rscript -e "require(mxnet); mxnet:::mxnet.export('R-package'); warnings()"
-	rm R-package/NAMESPACE
+	$(RM) R-package/NAMESPACE
```
Comment (ankkhedia) [inline Makefile]: Any reason for this change?

**Comment by ankkhedia** [comment]
@hetong007 @marcoabreu Could you please help review the PR?

File: Makefile (hunk 1 above)
Comment (shadogray) [reply Makefile]: for this Makefile a global remove command is defined, and it is reasonable to make consistent use of it

**Comment by karan6181** [comment]
@shadogray is the build still failing? I see no activities since Feb-15 on this PR. Can you please look into it and let us know if this PR needs to close it? Thanks!

**Comment by shadogray** [comment]
@karan6181 I would like to see this PR to be merged, it is not failing, as can be seen in the CI-build?
Can I in any way help to clarify the situation? I am most willing to do so, sorry if I misunderstood the process.

**Comment by abhinavs95** [comment]
@marcoabreu could you help @shadogray with the CI builds. Looks like the last commit was almost a month ago but the jobs are still pending.

**Comment by piyushghai** [comment]
@shadogray Can you push an empty commit to re-trigger the CI on your PR ? 

You can do : ```git commit -m "Empty Commit" --allow-empty
Then push this commit to this branch.

**Comment by Roshrini** [comment]
@shadogray Can you please resolve conflicts?

**Comment by shadogray** [comment]
@Roshrini  can you assist in finishing this seemingly trivial task?
I have synchronized now for the 4th time, only to see the complete build fail in some completely different area. The currently failing code is honestly totally unrelated to my changes. 
No rant intended, I highly appreciate this project.

**Pull request closed without merging**